packages/btfs-http-client/examples/upload-file-via-browser/config.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
probably need a `git rm` to get rid of this, future changes will be ignored

Reply:
okay. got it.